src/content/docs/iast/troubleshooting/unable-to-detect-vulnerabilities-for-golang-application.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: No vulnerabilities detected for Golang application
+tags:
+  - IAST
+  - New Relic IAST
+  - Troubleshooting
+metaDescription: Troubleshoot Golang application if no vulnerabilities are detected.
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+You don't see any vulnerability for your Golang application.
+
+## Solution
+
+- For Golang, ensure that you imported the required instrumentation packages of libraries and frameworks.
+
+    For example, your application is using libraries for [Mongo DB](http://go.mongodb.org/mongo-driver/mongo). For this specific library, you need to import this [instrumentation package](https://github.com/newrelic/csec-go-agent/tree/main/instrumentation/csec_mongodb_mongo) from newrelic:
+    
+    ```
+    import (
+        "github.com/newrelic/go-agent/v3/integrations/nrsecurityagent"
+        "github.com/newrelic/go-agent/v3/newrelic"
+        "github.com/newrelic/csec-go-agent/instrumentation/csec_mongodb_mongo"
+        "go.mongodb.org/mongo-driver/mongo"               
+    )
+    ```
+
+    Depending on you application requierement, you might have to import additional packages. Check [list of supported packages](https://github.com/newrelic/csec-go-agent#instrumentation-packages).
+
+    Also, Make sure you imported the corresponding [<InlinePopover type="apm" /> package](https://github.com/newrelic/go-agent?tab=readme-ov-file#service-frameworks) for each module.
+
+    - For HTTP protocol endpoint, use the following `newrelic.WrapListen` function to enable vulnerability scanning against the endpoint:
+
+    ```
+    http.ListenAndServe(newrelic.WrapListen(":<YOUR PORT NUMBER>"), nil)
+    ```
+
+    <Callout variant="important">
+        You can skip this step if you're on a Linux environment.
+    </Callout>
+
+    - For a specific case of <DNT>**Outbound HTTP Request**</DNT> or calls to external services, you need to [update your application method](/docs/apm/agents/go-agent/instrumentation/instrument-go-segments/#go-external-segments).
```

Comment:
I don't think we need to bold Outbound HTTP Request. Also, should that be plural?